### ISSUE TITLE
Handle environment variables in red.xml for classpath and pythonpaths

### ISSUE THREAD

**Issue by rubberballman** [nokia/RED#204]
Hi,
I would like to use a system environment variable to specify libraries path inside the classpath section of the red.xml file.
Something like this...
```
<classpath>
    <path location="${USER_HOME}/.m2/repository/org/robotframework/javalib-core/1.2.1/javalib-core-1.2.1.jar"/>
</classpath>
```
The idea is also to make sure the classpath will stay the same no matter which OS you're using (Windows, MacOS, Linux, etc.).
Any idea how I can achieve such a thing?
Thanks

**Comment by KrzysztofJozefowicz** [comment]
Hi
as current (0.8.4) there is no native way to handle env vars in red.xml apart of own script which would change classpath entry with preselected values. 
There is inconsistency in handling such cases in red.xml as in variable mappings we already are allowing to use env vars for values,this should be also handled for PYTHON/CLASS PATHS as well. 
Should not be much to work as in VarMappings we already are reading and resolving env vars.

Other similar case is one reported by internal user - relative paths setting which works only for PYTHON/CLASSPATHS but should also work for paths in referenced libraries (now paths are always relative to workspace and consist project name). 

This cases should be taken into sprints for rel 0.8.6 as we are in the middle of preparing 0.8.5,until than only way is to create custom script which would update classpaths outside of RED.

**Comment by modrz** [comment]
Fixed in 0.8.6
